app/src/main/java/com/asfoundation/wallet/verification/VerificationActivityNavigator.kt

```diff
@@ -22,7 +24,17 @@ class VerificationActivityNavigator(private val fragmentManager: FragmentManager
   }
 
   fun finish() {
+    activity.finish()
+  }
 
+  fun backPress() {
+    activity.onBackPressed()
   }
 
+  fun navigateToWalletVerificationIntroNoStack() {
+    for (i in 0 until fragmentManager.backStackEntryCount) {
+      fragmentManager.popBackStack()
+    }
```

Comment:
I didn't do this on purpose because there were reports that this didn't work in some cases and it actually was not meant to be used this way. If you see the docs for this method, popBackStack(null, ...) should actually only pop the top state, and not all.